tests/integration/components/bs-button-test.js

```diff
@@ -76,6 +77,11 @@ module('Integration | Component | bs-button', function(hooks) {
     assert.dom('button i').hasClass('fa-check');
   });
 
+  testBS4('button with outline property shows as outline', async function(assert) {
+    await render(hbs`{{bs-button type="primary" outline=true}}`);
+    assert.dom('button').hasClass('btn-outline-primary');
```

Comment:
Can you add another assertion here that it does *not* have the `btn-primary` class? (I know it's working right, but just to make sure it)

Reply:
I meant to add this to the existing test:

```suggestion
    assert.dom('button').hasClass('btn-outline-primary');
    assert.dom('button').doesNotHaveClass('btn-primary');
```